app/src/app.py

```diff
@@ -1,13 +1,27 @@
 from flask import Flask, render_template
+from flask_login import LoginManager
 from models import db, User
 
 
 app = Flask(__name__)
 
 # Configure and initalize database
 app.config['SQLALCHEMY_DATABASE_URI'] = 'sqlite:///site.db'
+
+login_manager = LoginManager()
+login_manager.login_view = 'auth.login'
+login_manager.init_app(app)
 db.init_app(app)
 
+# blueprint with authorization routes for the app
+from auth import auth as auth_blueprint
```

Comment:
Should move this to the top, just to keep imports organized at the top of the file